pkg/mcp/loader.go

```diff
@@ -190,13 +190,15 @@ func (sm *SessionManager) ensureDeployment(ctx context.Context, server ServerCon
 				return gmcp.ServerConfig{}, fmt.Errorf("failed to parse MCP server URL: %w", err)
 			}
 
+			fmt.Printf("Checking that MCP server URL %s is not a localhost URL\n", server.URL)
```

Comment:
is that for debug or do we want to log these actually?

Reply:
I was using these for debugging. I removed some, but forgot these. Thanks for catching.